website/public/js/controllers/menuCtrl.js

```diff
@@ -23,6 +23,8 @@ angular.module('habitrpg')
           return unallocatedValue;
         } else if (!(_.isEmpty(user.newMessages))) {
           return messageValue;
+        }  else if ( user.flags.bootedFromGroupNotifications.length > 0 ) {
```

Comment:
Our style is to not have spaces here, so `(user.flags.bootedFromGroupNotifications.length > 0)` instead of `( user.flags.bootedFromGroupNotifications.length > 0 )`